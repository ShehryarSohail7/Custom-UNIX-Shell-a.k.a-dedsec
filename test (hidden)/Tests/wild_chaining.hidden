mkdir test_dir && cp * test_dir ; ls ; ls test_dir
rm test_dir/* && rmdir test_dir ; ls ; ls test_dir
