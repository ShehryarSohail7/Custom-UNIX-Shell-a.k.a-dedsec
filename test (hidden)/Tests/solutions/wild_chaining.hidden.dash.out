&&
*
;
Tests
config.json
cp
find.log
ls
names.txt
test.py
test_dir
&&
*
;
Tests
config.json
cp
find.log
ls
names.txt
test.py
test_dir
